lib/lab_manager/provider/vsphere.rb

```diff
@@ -111,10 +111,12 @@ def create_vm(opts = {})
               ", vm_name: #{vm_name}"
           )
         end
+
         Retryable.retryable(
-          tries: 3,
-          on: [RbVmomi::Fault, CreateVMError],
-          exception_cb: exception_cb
+          tries: VSphereConfig.vm_clone_retry_count,
+          on: [RbVmomi::Fault, CreateVMError, Fog::Compute::Vsphere::NotFound],
+          exception_cb: exception_cb,
+          sleep: ->(n) { Random.rand(n*3..n*3+10.0) }
```

Comment:
The additional verbosity seems like bikeshedding to me. What do you think @h4rr0d, @ondrej-hosak, @zdenekN61?